salt/modules/dockermod.py

```diff
@@ -1945,7 +1945,6 @@ def resolve_tag(name, tags=None, **kwargs):
 
     tags
         .. deprecated:: 2018.3.0
-            Ignored if passed, will be removed in the Neon release.
```

Comment:
We can probably also remove the rest of the documentation for this argument, what do you think?

Reply:
I have done that in the past, but I was thinking it might be nice for people to know what happened to the option if they go looking. Thoughts?